### ISSUE TITLE
HIST scenario in Echam with incorrect greenhouse gases file

### ISSUE THREAD

**Issue by mandresm** [esm-tools/esm_tools#913]
Reported by @PengyangSong

`HIST` scenario in Echam uses the incorrect greenhouse gases file, and that means that for certain years in a historic simulation the files are not found. This is in contrast with the `cmip6hist` scenario, which is the option used by GEOMAR. A fix is needed for `HIST`.

**Comment by chrisdane** [comment]
Hi Miguel just fyi, for my work I had to make some changes in this hist-context, too. Here is the compare to release: https://github.com/esm-tools/esm_tools/compare/release...fix_echam_hist_ssp_forcing

I did not had the time to make sure it can be merged to release. Maybe it helps. 

historical experiment obtained with this branch on levante: 
```
/work/ab1095/a270073/out/awicm-1.0-recom/awi-esm-1-1-lr_kh800/historical3
```

Cheers,
Chris
